prime-router/metadata/hl7_mapping/ORU_R01/base/specimen.yml

```diff
@@ -143,6 +143,6 @@ elements:
     schema: observation-result
     resourceIndex: resultIndex
     constants:
-      hl7ObservationPath: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/SPECIMEN(%{resultIndex})'
-      hl7OBXField: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/SPECIMEN(%{resultIndex})/OBX'
+      hl7ObservationPath: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/SPECIMEN(%{resultIndex})/SPECIMEN_OBSERVATION'
+      hl7OBXField: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/SPECIMEN(%{resultIndex})/SPECIMEN_OBSERVATION(%{resultIndex})/OBX'
```

Comment:
Is `resultIndex` supposed to be in `hl7OBXField` twice? If so, can you explain that?

Reply:
Good catch...I need to look at this again.  I think this only works for our case of there only being a single observation under the specimen

Reply:
Updated